pkg/volume/iscsi/iscsi.go

```diff
@@ -56,16 +57,12 @@ func (plugin *iscsiPlugin) GetPluginName() string {
 }
 
 func (plugin *iscsiPlugin) GetVolumeName(spec *volume.Spec) (string, error) {
-	volumeSource, _, err := getVolumeSource(spec)
+	tp, iqn, lun, _, err := getISCSITargetInfo(spec)
```

Comment:
getISCSITargetInfo returns "target portal, portals, iqn, and lun".
I think this should be;
` tp, _, iqn, lun, err := getISCSITargetInfo(spec)`